modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,147 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify which business groups allow content to be exported to connections. You can also edit and delete connections and configurations.
+
+// add back to configs: "configuration to specify the types of data Anypoint Platform exports as well as..."
+
+
+== Before You Begin
+
+Before you configure the Telemetry Exporter feature, ensure that you have:
+
+* To create or modify connections, an Anypoint Platform account with the Telemetry Exporter Administrator permission
+* To create or modify configurations, an Anypoint Platform account with the Telemetry Exporter Configuration Manager permission
+* Access information for your third-party app, including the OpenTelemetry-supported endpoint. For example, the endpoint URL for Azure Monitor is `https://dc.services.visualstudio.com/v2/track`.
+* The ability to create an instrumentation key or access token for your third-party app
+
```

Comment:
// add pricing requirement

Reply:
These bullets need periods since you use one in line 22.